pom.xml

```diff
@@ -170,11 +170,6 @@
       <version>1.4.2</version>
       <type>pom</type>
     </dependency>
-    <dependency>
```

Comment:
Why was this removed?

Reply:
@SerhiiNosko please explain. thank you

Reply:
Data processing core had folio-kafka-wrapper with 3.0.0-SNAPSHOT version and folio-kafka-wrapper had 2.6.0 version that causes conflicts. Now folio-kafka-wrapper is getting as transitive dependency from data-import-processing-core with 3.0.0-SNAPHOT version